modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/GridUpdateNotifier.java

```diff
@@ -117,6 +94,9 @@
     /** Worker thread to process http request. */
     private final Thread workerThread;
 
+    /** Url for request version. */
+    private String  url = "https://ignite.run/update_status_ignite-plain-text.php";
```

Comment:
Seems this should be private final static UPDATE_NOTIFIER_URL
Also, please remove redundant space between String  url